language-support/qute/qute.tmLanguage.json

```diff
@@ -128,8 +131,11 @@
       "match": "((\\s+[^}\\/]+)*)\\s*",
       "name": "variable.parameter.function.qute"
     },
+    "double_bracket": {
+      "match": "{{"
+    },
     "expression": {
-      "begin": "((?<!\\\\){)(?![\\s!#@\\/])",
+      "begin": "((?<!\\\\){)(?=\\w)",
```

Comment:
It seems this can no longer match just `{}` brackets alone, but it will match the brackets when used with text: `{<text>}`